sdk/keyvault/Azure.ResourceManager.KeyVault/README.md

```diff
@@ -9,7 +9,7 @@ This package follows the [new Azure SDK guidelines](https://azure.github.io/azur
 Install the Azure Key Vault management library for .NET with [NuGet](https://www.nuget.org/):
 
 ```PowerShell
-dotnet install Azure.ResourceManager.KeyVault -Version 1.0.0-preview.2 
+dotnet add package Azure.ResourceManager.KeyVault -Version 1.0.0-preview.2
```

Comment:
```suggestion
dotnet add package Azure.ResourceManager.KeyVault -version 1.0.0-preview.2
```

Reply:
Actually, this has to be `--version` (two dashes).